app/src/main/kotlin/com/simplemobiletools/smsmessenger/dialogs/MessageDetailsDialog.kt

```diff
@@ -0,0 +1,72 @@
+package com.simplemobiletools.smsmessenger.dialogs
+
+import android.annotation.SuppressLint
+import android.telephony.SubscriptionInfo
+import com.simplemobiletools.commons.activities.BaseSimpleActivity
+import com.simplemobiletools.commons.extensions.getAlertDialogBuilder
+import com.simplemobiletools.commons.extensions.getTimeFormat
+import com.simplemobiletools.commons.extensions.setupDialogStuff
+import com.simplemobiletools.smsmessenger.R
+import com.simplemobiletools.smsmessenger.extensions.config
+import com.simplemobiletools.smsmessenger.extensions.subscriptionManagerCompat
+import com.simplemobiletools.smsmessenger.models.Message
+import kotlinx.android.synthetic.main.dialog_message_details.view.dialog_message_details_text_value
+import org.joda.time.DateTime
+
+class MessageDetailsDialog(val activity: BaseSimpleActivity, val message: Message) {
+    init {
+        @SuppressLint("MissingPermission")
+        val availableSIMs = activity.subscriptionManagerCompat().activeSubscriptionInfoList
+
+        @SuppressLint("SetTextI18n")
+        val view = activity.layoutInflater.inflate(R.layout.dialog_message_details, null).apply {
+            dialog_message_details_text_value.text = """
+                 ${activity.getString(R.string.message_details_type)}: ${message.getMessageType()}
+                 ${message.getReceiverOrSenderLabel()}: ${message.getReceiverOrSenderPhoneNumbers()}
+                 SIM: ${message.getSIM(availableSIMs)}
```

Comment:
lets show the SIM only if the user has multiple SIMs. The vast majority of users have 1 only, so we would show a useles thing.